lndclient/signer_client.go

```diff
@@ -14,6 +15,17 @@ import (
 type SignerClient interface {
 	SignOutputRaw(ctx context.Context, tx *wire.MsgTx,
 		signDescriptors []*input.SignDescriptor) ([][]byte, error)
+
+	// SignMessage signs a message with the key specified in the key
+	// locator. The returned signature is fixed-size LN wire format encoded.
+	SignMessage(ctx context.Context, msg []byte,
+		locator keychain.KeyLocator) ([]byte, error)
+
+	// VerifyMessage verifies a signature over a message using the public
+	// key provided. The signature must be fixed-size LN wire format
+	// encoded.
+	VerifyMessage(ctx context.Context, msg, sig, pubkey []byte) (bool,
```

Comment:
Nit: make `pubkey [33]byte` so that it's clear a compressed public key should be used.

Reply:
Yes, good idea. Fixed.